sources/install.sh

```diff
@@ -2125,6 +2125,13 @@ function install_pcredz() {
   add-test-command "PCredz --help"
 }
 
+function install_spray() {
+  # Password Spraying tool for Active Directory Credentials.
+  colorecho "Installing Spray"
+  git -C /opt/tools/ clone https://github.com/Greenwolf/Spray.git
+  add-aliases spray
+}
```

Comment:
Can you add `add-test-command`, as well as `add-history` if you have some example commands at mind?

Reply:
Hello @ShutdownRepo, normally all the changes you requested has been pushed